app/views/LocationTracking.js

```diff
@@ -41,6 +42,7 @@ import languages from '../locales/languages';
0tAe1hd9D17V0NZNKGLxSF/fhLnY9Ht6CH0N/Zm3nkLDnEty28boE3QjshuZF0PxbatiCsFn9jtVH0i4J8AieEyFMvmU9FYNALl+k/EyjfWs/XNTywb7b8XFaEI1IAAI34NFOvNNuOpy95J2LGZH2T+YpjxGdBx0HIgAjkI2PkI2dIEDucnO/QKaXYu6gw0jEMrewFqx9bwY8nP/IY+12V8jJNMBEQgJQH2HCYG3kWxvZP9Q7bR/KPQe4mTQrZBvkVABNoQYJAPQSFtHs73bBNGIYutHmT1hTS79UMmAlEI9I5SSwGVMOLs8HVtZHee9yrAZVcuDu1qQUG/H8+h27UF+WrpxuqB2/EU+mHLgp0tPJI6Lu7MhdZOROAt6n0FTWdbse/uLdcVqditaSSpz1KvzTM+HPWLHUPB9V3NhrFPwT7buoPj1RTau21BFagrgVk0/B50Jfol26j97dLcJiwG2S4QOxdV5ZDjVdoyiI1hRuwtAZZrUuejqH/sulVf6QjYq96+xnZ6mcfIXV4lZIB9A1jXo6okK+v776VIVlZxo96z7LtMBNoQsPdr/ooxeD5apk3Z6Ivd7WEByR6HOSU6ibAV2iM165E4ks1yANdViOFJVPbD6rA9Je+LE7iUbTb0Od3F62v73dUeFoNqPyKuWrKyTrgqZbKyABr1X2XfZSKQkcDnGZPHZCwbpZibhAWYFWjxeVFaHb+S38SvsmmNXuJoGpx+dEngO4zNVb1E5iZhAcQy+VpewBQYxzx83VGgv05cWRwWj0wEshJYiYInZC0cupynhPVvoRubyP9EDsfsHFZya8QxMXkgCqBsBOy2IhfmImGxy2nTo2zogkjxQUwp3mVHHr3F01FjtHIUAusyRj8UpaY2lbhIWMRYpdsXlkQe/b6rJQNY4m9v8SwRnv50SmCwh7i8JKz3e4ARKAYXh4OLtc1bPIuFpq+OCazmITYvCcvmCa+qeWG8iK+3eBbFpU/fBFw8a+hl433Wd191FN2KHa1d/Mre4im+hfIYgsAzIZzm9eklYT2UN/ASlV/fWaze4nGGR+E0IWCTQT7Y5PfoP7lIWFxutz0se51WFW2Is0Z5i8cZHoXThMB4xuhzTX6P/pOLhNVo9c+jtz5OhetzSXjdOFW1rqURx/qtS2mpCCxF4Pylfkn0g6eEdSEMpiXiELraT4WuIKN/L3FkDFfFHBB4mBgucRDHghDcJCx2OecQ0ZdQFa8Yft5Jh7t68t4JE4XRNYHZLDqcselmTLpJWMYMMDYH1on2vWK2HYdjW6ZsE/VvRf3bpoxBdZeKgCWpwxiTf/YUtauEZWAAdDYfX0S2x1UlOy1xY05NXL+qLw+BmYS6J2NxtLeQ3SUsAwSo/+HjY8j2uKpie7CXMzJFY6h3FPXukaJu1VkqAnb7wuVoc8bgdR4jdzfj6JKQGGxb85u9w2979BG0MnKZaImrnT1NgS3YGP7ZrmBRy+G3Br4eQDavu0wEFidgh302C+4jaBwazbY5mU9ZWQgwwI9BIe02nNtbk4Ob1YPGoZB2bPCGqAIREIHmBBjZa6P5IUc4vq9CQd8Jaf4b9fARzIzTOs1J6lcREIEoBBiEdwYb4u86vp6v/UM0yPwi8x/a7g4Rv3yKQFcEynouqKv2FPX7pUU5auFnN5aNJ6MMbVEm96KGvwmsaP5DWwxOodsg/yJQbgIM+r7oJRTD5lLJuchew9Vts/UbfsxfDHuZSvTKsG73mFYUgQIJMBgticS0mVT2fTQ4TzMoP6Sxnq0f036QJ06VFYEiCLi/raGIRnbHByP/g6w3BUW5ordEjPaiiFuRHdpZDM+jN5C9Cu39aENkd87vjFJMLz3XYuAS+DQ+ZSIQjYASVgvUJK0fsfjLLYrUddFPSVZH17Xxanc6AkpYLdiTsD7A4seR7dnIFhJ4k48Pk7CeFhARiE1AVwlbEGdQ2rSwp7coUsdFZyhZ1bHbfbRZe1ht+oG9rGUoYo+2aKbOHj3ssQ17zqxqD6a32Qq02AsB7WG16QkGp51gPgK5mROoTcihFlv7j1SyCoVXfrMQUMLKQIlBehfF/itD0SoX+W843FHlBqpt/gnokDBjH3Fo2IuidqvB8IyrVKnYnTRmBAnLxbvpqgRWbclHQAkrBy+S1loUvxfV6YHf6bR3GMlKVwVzbCsqGoaADglzcG0M2k+zis3IWAd7lUbuoWRVh64uRxuVsHL2E4P3IVbZF9kE/VU2uxK4L+2t6vsiq9x3lW2bElY3upZBfBOr7YVmdWP1MqxiyXhv2nljGYJVjPUhoHNYHfQ157R2YfVr0fIduPG2qt3JbsnqBm+BKR4RUMLqcBsgadnrsyxp2Qn5stsMGmBvS7mv7A1R/NUkoIRVQL+StNbGzRhU6GR8BYSWx8X9FB5FsrKrgjIRcElA57AK6BYG+VO42Radh+xVSWUyi9fmtvq4klWZuk2xikABBNjb2g1NR2Uwi3P3ApotFyIgAmUlQBLoh76H5iCPZnGdhTTFcVk3MsUtAkUTICFshK5EoV8bRhWZzOL4Lco1DXPRXORPBETAMQESxMbo1yjVHpfVa/WnmE7Zcc8oNBEQgS4JkDAGoJPQVBTDrB6rb0CXQWmBCJSIgG5rSNRZJJFNqXpkQ/ZCid4FhDIPH39BY9EYrvrpsZoCoMqFHwJKWA76guRlc8bbPVzDkM1sOrChNflcDi1p9uiM3eT5REM2E6jNIjGBJGVv15GJQCUJKGE571aSmc3D1Rf1Qfbs4uskJc1LBQiZCIiACIiACIiACIiACIiACIiACIiACIiACIiACIiACIiACIiACIiACIiACIiACIiACIiACIiACIiACIiACIiACIiACIiACIiACIiACIiACIiACIiACIiACIiACIiACIiACIiACIiACIiACIiACIiACIiACIiACIiACIiACIiACIiACIiACIiACIiACIiACIiACIhACQn8P8rOOhTDbYOjAAAAAElFTkSuQmCC"
```

Comment:
Doesn't really belong here  ... at a minimum should be over in the images directory.  Is it not possible to do the svg render on iOS 12.0?

Reply:
Yeah. As I have commented on the issue 528. It is issue with svg library which we are using. So I have used data instead of svg xml.

Reply:
I have moved image data to separate files inside assets. Please verify it.